### PR TITLE
I#261: Added Terms of Service and Privacy Policy Links functionality

### LINKED ISSUE #261: Add Terms of Service and Privacy Policy Links
Include links to the **Terms of Service** and **Privacy Policy**, and create corresponding content for transparency and user compliance.  

**Tasks**:  
- [x] Content for the **Terms of Service** outlining user responsibilities, platform rules, and liability.  
- [x] Content for the **Privacy Policy** explaining data usage, storage, and sharing practices.  
- [x] Add a "Terms of Service" link to the footer or relevant sections of the app.  
- [x] Add a "Privacy Policy" link alongside the Terms of Service link.  
- [x] Ensure the links open in a new tab of the user's default browser without disrupting the user flow.

**Acceptance Criteria**:  
- [x] **Terms of Service** and **Privacy Policy** are created.  
- [x] Clicking the links opens the respective documents in a new tab of the user's default browser.

### PR DESCRIPTION
**Pull request by Shahed-Chy-Suzan**
Fixes #261

- [x] Does not depend on other PRs

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [x] New feature
- [ ] Refactor
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?

Added Terms of Service and Privacy Policy links functionality to the adding personal information page, enabling users to view legal documents directly from the app using device default browser.

## How to test

- [x] Needs regression testing (add QA needed label)

Test steps:
1. Navigate to the personal information page
2. Verify Terms of Service and Privacy Policy links are visible
3. Tap on each link and confirm they open the correct URLs
4. Ensure links are functional and redirect to the correct documents

## Checklist

- [x] I have tested these changes locally and they function as expected.
- [x] I have added or updated necessary documentation to reflect the changes.
- [x] I have followed the project's coding style and guidelines.
- [ ] I have added appropriate test coverage to ensure the changes are functioning correctly.
- [x] I have checked for any potential conflicts with other branches or pull requests.
- [ ] I have updated necessary OpenAPI specs

## Additional Notes (optional)

The Terms of Service and Privacy Policy links are now pointing to static HTML pages from our BloodConnect project's documentaion hosted on the project's GitHub Pages.

## Deploy risk & follow up

- [x] No risk

### CODE REVIEW

File: docs/legal/TermsOfService.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
+======================
+Terms and Conditions
```
Comment (sakibstark11) [inline docs/legal/TermsOfService.rst:2]: assuming this has been reviewed by a legal expert of some sort?

File: docs/legal/TermsOfService.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+If you do not agree to all the Terms and Conditions stated herein, you are prohibited from using the platform.
+
+1. About BloodConnect
+----------------------
+BloodConnect is an open-source platform aimed at improving the blood donation process by facilitating efficient and effective communication between donors and blood seekers. The platform prioritizes ease of use, data security, and rapid communication to address the challenges of manual donor identification and coordination in emergency situations.
+
+2. Open Source Usage
+--------------------
+BloodConnect is an open-source project licensed under the [Insert License Name, e.g., MIT License]. As an open-source project:
```
Comment (sakibstark11) [inline docs/legal/TermsOfService.rst:19]: ?

File: docs/legal/TermsOfService.rst (hunk 1 above)
Comment (Shahed-Chy-Suzan) [reply docs/legal/TermsOfService.rst:2]: Khirul vai is reviewing it, his review is still going on.

File: docs/legal/TermsOfService.rst (hunk 2 above)
Comment (Shahed-Chy-Suzan) [reply docs/legal/TermsOfService.rst]: updated at 6cdd6180d936740b721200a4e055c9c990763be1

**Review by mahinuzzaman — APPROVED** [review]

**Pull request merged**